defaults/main.yml

```diff
@@ -25,6 +25,7 @@ nginx_keepalive_requests: 100000
 nginx_client_max_body_size: 50M
 
 nginx_limit_no_file: 1024
+nginx_private_tmp: true
```

Comment:
Please, also add nginx_private_tmp: false var in molecule tests.
And don't forget modify the CHANGELOG file :)